Tools/ssl/multissltests.py

```diff
@@ -70,9 +74,11 @@
 parser = argparse.ArgumentParser(
     prog='multissl',
     description=(
-        "Run CPython tests with multiple OpenSSL and LibreSSL "
+        "Run CPython tests with multiple cryptography libraries"
         "versions."
-    )
+    ),
+    color=True,
+    suggest_on_error=True,
```

Comment:
These seem to be causing us some issues:

```suggestion
    **(
        dict(
            color=True,
            suggest_on_error=True,
        ) if sys.version_info >= (3, 14) else {}
    )
```

Or, to simplify:

```suggestion
```

Considering the comment on line 21, the latter is probably the better option.

Reply:
Thanks, I'll remove the coloration directives for now.

This was [suggested](https://github.com/python/cpython/pull/135402#discussion_r2190411684) by @hugovk -- if he prefers, I'll revert and use @zware's kwarg-dict-unpacking approach.

Reply:
Ah right, revert is the way to go, thanks.

It's because this script is run before we've built our own Python and we're using image-default one, likely version 3.13.

https://github.com/python/cpython/actions/runs/16229017511/job/45827626323?pr=135402#step:9:1

We can instead do like `parser.color = True`, but it can be for the refactor.

https://docs.python.org/3.14/library/argparse.html#color